app/models/search_result.rb

```diff
@@ -0,0 +1,39 @@
+class SearchResult
```

Comment:
should this live under `view_models` as it is a model built from the JSON response?